### DISCUSSION TITLE
[Released] Github style Team/Org level ENVs?

### DISCUSSION THREAD

**Discussion by thihathit** [vercel/community#1008] [Idea]
It would be cool to have Github's org secret style ENVs on Vercel dashboard.

The most painful thing is having to set variables like `NPM_RC` for each project, even though these values are always the same. And when we need to swap them out, we have to update by each project, which isn't very nice experience.

**Comment by amyegan** [comment]
Great idea! The team is already working on a feature to address this 😁

**Reply by amyegan** [reply]
You can now use [Shared Environment Variables](https://vercel.com/changelog/share-environment-variables-across-your-team-and-projects) across team projects! This feature is currently available to all Teams on the Pro and Enterprise plans.

**Reply by thihathit** [reply]
awesome! 🎉

**Comment by thihathit** [comment]
Will the `input` gonna become `textarea` instead? just like the ones from instance.

Expected:

![image](https://user-images.githubusercontent.com/30023628/200766480-9f0d1171-58fb-4c46-9b69-83f76e32f943.png)

Current:

![image](https://user-images.githubusercontent.com/30023628/200766549-e84c52bc-a321-4c1d-90f1-0882c1b6288b.png)

**Reply by thihathit** [reply]
text input doesn't accept line-breaks

**Reply by thihathit** [reply]
@amyegan, That is breaking builds currently.

this is my use-case

`NPM_RC`
```
//npm.pkg.github.com/:_authToken=xxxx

@org-name:registry=https://npm.pkg.github.com/
```

**Reply by amyegan** [reply]
I'm sharing this feedback to the team now and I, or someone from the engineering team, will share an update ASAP!

**Reply by amyegan** [reply]
The team just let me know that they are already working on it 🙏
